.travis.yml

```diff
@@ -39,4 +39,5 @@ script:
     - ./test.sh --nologcapture
     - ./lint.sh
     - ../node_modules/.bin/mocha static/build_test.js
+    - docker run --rm=true -i -w "/go/src/k8s.io/test-infra" -v "${GOPATH}/src/k8s.io/test-infra:/go/src/k8s.io/test-infra" 'gcr.io/google_containers/kubekins-job-builder:5' bash -c 'jenkins-jobs --ignore-cache test -o output jenkins/job-configs:jenkins/job-configs/kubernetes-jenkins'
```

Comment:
Please remove the `-i` as travis does not interact with the container via stdin

Also this is a very complicated command. Any chance you can put it into a script and explain what the various pieces are doing?